assets/scripts/modules/Load.js

```diff
@@ -1,22 +1,35 @@
 import { module } from 'modujs';
 import modularLoad from 'modularload';
+import { resetLazyloadCallbacks, triggerLazyloadCallbacks } from "../utils/image";
 
 export default class extends module {
     constructor(m) {
         super(m);
     }
 
     init() {
-        const load = new modularLoad({
+        this.load = new modularLoad({
             enterDelay: 0,
             transitions: {
                 customTransition: {}
             }
         });
 
-        load.on('loaded', (transition, oldContainer, newContainer) => {
+        this.load.on('loaded', (transition, oldContainer, newContainer) => {
             this.call('destroy', oldContainer, 'app');
             this.call('update', newContainer, 'app');
+
+            /**
+             * Trigger lazyload
+             */
+            triggerLazyloadCallbacks();
+        });
+
+        this.load.on("loading", () => {
```

Comment:
Respect the existing quotation style and prioritize single quotes.

```suggestion
        this.load.on('loading', () => {
```